src/pathfinding_dijikstra.cpp

```diff
@@ -567,15 +578,18 @@ inline DijikstraPathfinding::ExpansionOutcome DijikstraPathfinding::expand_2d_up
         }
     }
 
-    this->domain = route_settings.is_relative_search_domain() ? MapDomain::RELATIVE_DOMAIN :
+    this->domain = route_settings.is_relative_search_domain() ?
+                   MapDomain::RELATIVE_DOMAIN :
                    MapDomain::ABSOLUTE_DOMAIN;
 
     if( result == ExpansionOutcome::UNSET ) {
         if( is_fully_explored ) {
             // Map must have no path to target
-            for( size_t i = 0; i < DIJIKSTRA_ARRAY_SIZE; i++ ) {
-                if( is_nan( this->d_map.g[i] ) ) {
-                    this->d_map.p[i] = NAN;
+            for( size_t y = 0; y < MAPSIZE_Y; y++ ) {
+                for( size_t x = 0; x < MAPSIZE_X; x++ ) {
+                    if( this->d_map.tile_state[y][x] == DijikstraMap::State::UNVISITED ) {
+                        this->d_map.tile_state[y][x] == DijikstraMap::State::INACCESSIBLE;
```

Comment:
this is a comparison, did you mean 
```suggestion
                        this->d_map.tile_state[y][x] = DijikstraMap::State::INACCESSIBLE;
```
?